src/validator/CreditCardVendor.java

```diff
@@ -0,0 +1,32 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardVendor {
+	private String name;
+	private ArrayList<Integer> numberOfChars;
+	private ArrayList<String> IIN;
```

Comment:
It should not be IIN but rather masks. All fields and methods should use this instead of IIN. This is easier to understand at a high level.